src/pybind/mgr/dashboard/frontend/src/app/core/navigation/navigation/navigation.component.html

```diff
@@ -156,7 +156,11 @@
             class="tc_submenuitem tc_submenuitem_monitoring"
             *ngIf="permissions.prometheus.read">
           <a i18n
-             routerLink="/monitoring">Monitoring</a>
+             routerLink="/monitoring">
+            <ng-container i18n>Monitoring</ng-container>
+            <small *ngIf="prometheusAlertService.alerts.length > 0"
```

Comment:
If Prometheus is not enabled you get the following error `Uncaught TypeError: Cannot read property 'alerts' of undefined`.

Reply:
Can not reproduce this. I also wonder because if Prometheus is disabled, the service property `alerts` is an empty array and always exists.

Reply:
Hi retested it and found out that something went wrong within the node server that is spawned during `npm start` after restarting it the error was gone.